libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/location/replay/ReplayLocationDto.java

```diff
@@ -0,0 +1,90 @@
+package com.mapbox.services.android.navigation.v5.location.replay;
+
+import com.google.gson.annotations.JsonAdapter;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Date;
+
+class ReplayLocationDto {
+
+  @SerializedName("lng")
+  private double longitude;
+  @SerializedName("horizontalAccuracy")
+  private float horizontalAccuracyMeters;
+  @SerializedName("course")
+  private double bearing;
+  @SerializedName("verticalAccuracy")
+  private float verticalAccuracyMeters;
+  private double speed;
+  @SerializedName("lat")
+  private double latitude;
+  @SerializedName("altitude")
+  private double altitude;
+  @SerializedName("timestamp")
+  @JsonAdapter(TimestampAdapter.class)
+  private Date time;
+
+  public double getLongitude() {
```

Comment:
Should the getters / setters `public` visibility modifier be removed here as well?  Because the class `ReplayLocationDto` is package protected.